node-nut.js

```diff
@@ -29,8 +30,13 @@ Nut.prototype.start = function () {
 	this._client.setEncoding('ascii');
 
 	this._client.on('data', function (data) {
+		self.dataInBuff += data;
+		if(self.dataInBuff.slice(-1) != "\n") {
```

Comment:
This lines could be better with a space after `if`, a strict comparision and single quotes:
```javascript
if (self.dataInBuff.slice(-1) !== '\n') {
```